docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -397,6 +397,62 @@ To better understand the snap connections, read the [interface management](https
     sudo snap restart edgexfoundry.security-secretstore-setup
     ```
 
+<!-- DO NOT CHANGE THE TITLE. There are reference to the anchor -->
+#### Add-on services
+The platform is pre-configured with a set of [add-on services](../../security/Ch-Configuring-Add-On-Services/).
+
+The equivalent snap options for addon services are:
+
+- `apps.security-secretstore-setup.config.add-secretstore-tokens`
+- `apps.security-secretstore-setup.config.add-known-secrets`
+- `apps.security-bootstrapper.config.add-registry-acl-roles`
+
+!!! example "Query Secret Store tokens"
+    ```bash
+    sudo snap get edgexfoundry apps.security-secretstore-setup.config.add-secretstore-tokens
+    ```
+    Outputs: `app-functional-tests,app-rules-engine,app-http-export,...`
+
+Adding a new service is possible by appending its name to the value of the above options.
+
+!!! example "Add an add-on service with known secrets and Consul ACL role"
+    First, enable the new snap options, if they aren't:
+    
+    ```bash
+    sudo snap set edgexfoundry app-options=true
+    ```
+    
+    For each option, we need to:
+    
+    1. Query the existing items
+    2. Append the new service to the existing items
+    3. Set the appended string
+
+    ```bash
+    SERVICE="my-new-service"
+
+    # Secret store token
+    EXISTING=$(snap get edgexfoundry apps.security-secretstore-setup.config.add-secretstore-tokens)
```

Comment:
There needs a `sudo`, otherwise, some users will get access denied error, and get an empty $EXISTING, which causes an error:
```
$ sudo snap start edgexfoundry.security-secretstore-setup # for creating new tokens
error: cannot perform the following tasks:
- Run service command "start" for services ["security-secretstore-setup"] of snap "edgexfoundry" (systemctl command [start snap.edgexfoundry.security-secretstore-setup.service] failed with exit status 1: Job for snap.edgexfoundry.security-secretstore-setup.service failed because the control process exited with error code.
See "systemctl status snap.edgexfoundry.security-secretstore-setup.service" and "journalctl -xe" for details.
)
```